synapse/axon.py

```diff
@@ -1531,19 +1533,20 @@ async def postfiles(self, fields, url, params=None, headers=None, method='POST',
 
             except Exception as e:
```

Comment:
It looks like there are still differences with the exception handling in other handlers like wget.  Were those excluded intentionally?

Reply:
The normal use of the `postfiles()` API response is going to be wrapped into the Stormtypes HTTPResp object. The data currently returned does not map into what that object expects, so a `$lib.inet.http.*` user can get an object which doesn't have the expected error data present.

Since the remainder of the Axon APIs don't get wrapped in into a HTTPResp object, so changing the API responses for them would be a clear compatibility break for existing code which checks + uses the error data.

Reply:
I more thinking the inclusion of the err and reason keys, which wouldn't be backward incompatible.

Reply:
Ah - okay